internal/backup/archiver.go

```
@@ -728,6 +724,11 @@ func (a *Archiver) attachStderrLogger(cmd *exec.Cmd, algo string) error {
 	return nil
 }
 
+func drainTarWriterAfterCompressorStartFailure(pw *io.PipeWriter, errChan <-chan error, startErr error) {
+	_ = pw.CloseWithError(startErr)
+	_ = <-errChan
```

golangci-lint (2.12.2)

[error] S1005: unnecessary assignment to the blank identifier

(staticcheck)